syntax = "proto3";

package leaftest;

service ServerTest {
    rpc GetServerTime (TimeRequest) returns (TimeResponse) {}
}

message TimeRequest {
    // Empty request
}

message TimeResponse {
    int64 server_time_ms = 1;  // Server time in milliseconds
} 